s2wav/s2let_math.py

```diff
@@ -0,0 +1,83 @@
+"""Compute basic math tiling. """
+import numpy as np
+
+
+def f_s2dw(k, B):
+    """Tiling function for S2DW wavelets"""
+    t = (k - (1 / B)) * (2.0 * B / (B - 1)) - 1
+
+    return np.exp(-2.0 / (1.0 - t**2.0)) / k
+
+
+
+def f_needlet(t):
```

Comment:
Let's just support our wavelets (e.g. what was intially called s2dw).  So we can remove the needlet and spline cases.